extra-dev/packages/coq-serapi/coq-serapi.8.14+rc1+0.14.0/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer:   "e@x80.org"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "GPL-3.0-or-later"
+doc:          "https://ejgallego.github.io/coq-serapi/"
+
+synopsis:     "Serialization library and protocol for machine interaction with the Coq proof assistant"
+description:  """
+SerAPI is a library for machine-to-machine interaction with the
+Coq proof assistant, with particular emphasis on applications in IDEs,
+code analysis tools, and machine learning. SerAPI provides automatic
+serialization of Coq's internal OCaml datatypes from/to JSON or
+S-expressions (sexps).
+"""
+
+authors: [
+  "Emilio Jesús Gallego Arias"
+  "Karl Palmskog"
+  "Clément Pit-Claudel"
+  "Kaiyu Yang"
+]
+
+depends: [
+  "ocaml"               {           >= "4.07.0"              }
+  "coq"                 {           >= "8.14+rc1" & < "8.15" }
```

Comment:
I thought you want to put a stronger constraint here `= "8.14+rc1"`. If that's not the case, what's the point of making the `+rc1` explicit?

Reply:
Based on the discussion in the SerAPI PR, my understanding is that Emilio doesn't want this version to ever reach the regular opam repo, so we might as well use a wider range for convenience. I can change to `>= "8.14"` I guess.

Reply:
Yup, I guess any approach will work; usually we try indeed for serapi to work with all minor releases.